### DISCUSSION TITLE
Welcome to crdt-event-fold Discussions!

### DISCUSSION THREAD

**Discussion by rickowens** [owensmurray/crdt-event-fold#11] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rickowens** [comment]
@alaendle I see that you added `crdt-event-fold` (along with some of my other packages) to stackage! I'd be really curious to know what you're using it for.

**Reply by alaendle** [reply]
Hi! Thanks for reaching out. I actually added crdt-event-fold to Stackage just because they looked really well-designed and beautiful - I haven’t used it in any of my own projects yet. I hope it wasn’t presumptuous of me to add your package! I did this as part of my role as a Stackage curator, simply to help keep the package index relevant and up-to-date. If you’d prefer to maintain or transfer ownership of the Stackage listing, or if you’d like me to remove it, just let me know (or create a corresponding stackage PR).
